src/shm_sub.c

```diff
@@ -189,8 +189,11 @@ sr_shmsub_unlink(const char *name, const char *suffix1, int64_t suffix2)
 
     /* unlink */
     if (unlink(path) == -1) {
-        sr_errinfo_new(&err_info, SR_ERR_SYS, "Failed to unlink \"%s\" SHM (%s).", path, strerror(errno));
-        goto cleanup;
+        if (errno != ENOENT) {
```

Comment:
Please put the additional condition together with the current one, no need for a nested indent. Also, include the reason for the error being ignored.